pkgs/development/python-modules/ecos/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, pythonOlder
+, buildPythonPackage
+, fetchgit
+, numpy
+, scipy
+  # check inputs
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "ecos";
+  version = "2.0.7.post1";
+
+  disabled = pythonOlder "3.6";
+
+  # needs ecos submodule, so must be fetchgit vs fetchFromGitHub
+  src = fetchgit {
```

Comment:
fetchFromGit can also fetchsubmodules, although, under the covers it defaults to using fetchgit

Reply:
thanks.